FEM/Output.cpp

```diff
@@ -1421,6 +1461,29 @@ void COutput::WriteTECHeader(fstream& tec_file, int e_type, string e_type_name)
     }
 }
 
+bool COutput::open_tec_file(std::string tec_file_name,fstream& tec_file) const
```

Comment:
suggest changing `std::string tec_file_name` to `std::string const& tec_file_name` to avoid string copy

Reply:
Thanks for the hints!